src/test/shared/sam/cli/samCliInfo.test.ts

```diff
@@ -70,15 +70,15 @@ describe('SamCliInfoInvocation', async () => {
         const invoker: TestSamCliProcessInvoker = new TestSamCliProcessInvoker(
             (spawnOptions, args: any[]): ChildProcessResult => {
                 return new FakeChildProcessResult({
-                    stdout: '{"version": "1.2.3"}'
+                    stdout: '{"version": "999.999.999"}'
                 })
             }
         )
         const samInfo: SamCliInfoInvocation = new SamCliInfoInvocation(invoker)
 
         const response = await samInfo.execute()
         assert.ok(response)
-        assert.strictEqual(response.version, '1.2.3', 'unexpected sam info version')
+        assert.strictEqual(response.version, '999.999.999', 'unexpected sam info version')
```

Comment:
Not sure why this was changed as part of this PR. I don't think this is a validation test?

Reply:
That is correct but I noticed that 1.2.3 was used for version which is really bad for tests because that is an easy version number to hit, so changed that as part of this PR as well.

Reply:
This test doesn't test thresholds, though...it only tests to make sure a version is reported correctly.

Reply:
talked about offline, these look way too real IMO but reverted

Reply:
We can definitely look to change them, just not in this PR :)

Summary is: none of this code tests validation. We do have [explicit tests for validation](https://github.com/aws/aws-toolkit-vscode/blob/master/src/test/shared/sam/cli/samCliValidator.test.ts) and integration tests that do run SAM commands (and thus do test against the current SAM versions).